test/TensorFlowNET.UnitTest/NativeAPI/Eager/CApi.Eager.OpGetInputAndOutputLengths.cs

```diff
@@ -18,7 +18,7 @@ public unsafe void OpGetInputAndOutputLengths()
             var opts = TFE_NewContextOptions();
             using var ctx = TFE_NewContext(opts, status);
             CHECK_EQ(TF_OK, TF_GetCode(status), TF_Message(status));
-            TFE_DeleteContextOptions(opts);
+            opts.Dispose();
```

Comment:
📝 This is the second pattern. While the diff is smaller, it has a downside of only allowing the use of one `using` statement, which I've placed on `ctx` instead of `opts`.